.github/workflows/issue-triage.yml

```diff
@@ -79,6 +79,8 @@ jobs:
 
   issue-accepted:
     runs-on: ubuntu-22.04
+    permissions:
+      issues: write
```

Comment:
Is this the right way to go about it? It's needed for the action to have permission to remove the label, but I wasn't sure if this permission should be added to the GH token instead.